lib/src/widgets/utils/ds_card.widget.dart

```diff
@@ -61,15 +61,19 @@ class DSCard extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return _resolveWidget();
+    return _resolveWidget(messageType: type);
```

Comment:
If you followed my other suggestion, you could remove this ```_resolveWidget``` method and put it's code directly inside ```build```.